### ISSUE TITLE
Documentation not clear in `xyzSile.read_geometry`

### ISSUE THREAD

**Issue by pfebrer** [zerothi/sisl#569]
I didn't remember how to read multiple geoemtries from an `xyz` file, so I went to the documentation (https://zerothi.github.io/sisl/api/io/generated/sisl.io.xyzSile.html#sisl.io.xyzSile.read_geometry). In there, there is no mention at all that you can read multiple geometries, because the functionality is implemented in a decorator.

I think we should find some way of adding this to the documentation because otherwise nobody is going to use it.

(even going to the source code of the method does not clarify usage, you have to go to the source code of the decorator).

**Comment by pfebrer** [comment]
The function that appears on the docs is the original one or the one returned by the decorator?

**Comment by zerothi** [comment]
> The function that appears on the docs is the original one or the one returned by the decorator?

The decorator does not change the docs currently, so it is the wrapped functions doc-string.

**Comment by pfebrer** [comment]
Yes, but I mean what sphinx catches is the wrapped function, not the raw text of the function definition?

**Comment by zerothi** [comment]
I think sphinx uses the `func.__doc__` after import, to catch changes done automatically. Since the wrapped function uses `functools.wrap` it will retain the doc string.
